service/auth/jwt/rsa.go

```diff
@@ -0,0 +1,30 @@
+package jwt
+
+import (
+	"crypto/rsa"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/golang-jwt/jwt"
+)
+
+func getCertName(path, version string) string {
+	return fmt.Sprintf("%v/private_key%v.pem", path, version)
```

Comment:
use filepath.join instead

Reply:
This is not only filepath but also a version of certificate to use. Is better to read.

Reply:
??

Reply:
Now updated to filepath.

Reply:
The idea was to use [filepath.Join](https://pkg.go.dev/path/filepath#Join) instead of `fmt.Sprintf`.